aip/general/0133.md

```diff
@@ -194,17 +175,32 @@ NOT_FOUND errors][permission-denied].
 - For ensuring idempotency in `Create` methods, see [AIP-155][].
 - For naming resources involving Unicode, see [AIP-210][].
 
+## Rationale
+
+### Requiring user-settable ids
```

Comment:
I like to say "client-settable", which is more general than "user". It could be generated by a client, such as a controller.

Reply:
user-settable was the original nomenclature so I was sticking to that, but I personally also like the term "client" a bit better. will change.